exercises/practice/triangle/triangle_tests.plt

```diff
@@ -21,6 +21,9 @@ pending :-
     test(all_sides_are_floats_and_equal, condition(pending)) :-
         triangle((0.5), (0.5), (0.5), "equilateral").
 
+    test(all_sides_are_equal_but_mixed_types, condition(true)) :-
+        triangle(2, 2.0, 2, "equilateral").
```

Comment:
Could we make this something like `condition(bonus)` somehow?

Reply:
Please check this https://github.com/exercism/prolog/pull/289/commits/15654211aa346fc0c37762bf4fb03a10224341f6
Hopefully it addresses your request.